iniparse/ini.py

```diff
@@ -282,10 +282,15 @@ def set_value(self, data):
             else:
                 self.add(EmptyLine())
 
+    def get_line_number(self):
+       return self.contents[0].line_number if self.contents else None
```

Comment:
Will contents always be an array and will the item at the first index always have a line number attribute?

Reply:
It is defined above as self.contents = [], so it is always an array.
If I read the code correctly, contents is populated through the `add` method whose argument seems to always be a child class of LineType.